### PR TITLE
Added more page options and changed the default to match the initial …

### PR DESCRIPTION
**Pull request by jbolles94**
…screen height

### CODE REVIEW

File: src/packing_queue/tables/PackingQueueTable.jsx [hunk 1]
```diff
@@ -459,8 +464,16 @@ const PackingQueueTable = ({
           <tr>
             <TablePagination
               count={queueData.length}
-              rowsPerPageOptions={[numRowsPerPage]}
+              rowsPerPageOptions={PAGINATION_SIZING_OPTIONS}
               rowsPerPage={numRowsPerPage}
+              onRowsPerPageChange={(event) => {
```
Comment (joshplumley) [inline src/packing_queue/tables/PackingQueueTable.jsx:488]: This logic is repeated alot, should we make a little helper fn?

File: src/receiving_queue/tables/ReceivingHistoryTable.jsx [hunk 2]
```diff
@@ -182,6 +190,37 @@ const ReceivingHistoryTable = ({
     }
   };
 
+  const handlePageChange = (event, newPage) => {
```
Comment (joshplumley) [inline src/receiving_queue/tables/ReceivingHistoryTable.jsx:193]: should this be a useCallback

File: src/receiving_queue/tables/ReceivingHistoryTable.jsx [hunk 3]
```diff
@@ -182,6 +190,37 @@ const ReceivingHistoryTable = ({
...
+    setPage(newPage);
+  };
+
+  const generateTablePagination = useCallback(() => {
```
Comment (joshplumley) [inline src/receiving_queue/tables/ReceivingHistoryTable.jsx:197]: should this be a usecallback

File: src/packing_queue/tables/PackingQueueTable.jsx (hunk 1 above)
Comment (jbolles94) [reply src/packing_queue/tables/PackingQueueTable.jsx:489]: Reduced redundancy

File: src/packing_queue/PackingQueue.jsx [hunk 4]
```diff
@@ -50,7 +50,10 @@ const PackingQueue = () => {
   const [historyLoading, setHistoryLoading] = useState(false);
   const [histTotalCount, setHistTotalCount] = useState(0);
   const [histPageNum, setHistPageNum] = useState(0);
-  const histResultsPerPage = 10;
+  const [histResultsPerPage, setHistResultsPerPage] = useLocalStorage(
+    "packingHistNumRows",
+    window.innerHeight > 1440 ? 25 : 10
+  );
 
   // const [isShowUnfinishedBatches, setIsShowUnfinishedBatches] = useState(true);
```
[the remark this answers is not in the document]
Comment (jbolles94) [reply src/packing_queue/PackingQueue.jsx]: removed

File: src/packing_queue/tables/HistoryTable.jsx [hunk 5]
```diff
@@ -391,7 +397,22 @@ const HistoryTable = ({
...
+        rowsPerPageOptions={PAGINATION_SIZING_OPTIONS}
+        onPageSizeChange={(newPageSize) => {
+          onPageSizeChange(
+            newPageSize,
+            pageNumber,
+            filteredHist.length,
+            onPageChange,
+            setHistResultsPerPage
+          );
+          // const pageValue = newPageSize;
```
Comment (joshplumley) [inline src/packing_queue/tables/HistoryTable.jsx:409]: comment

**Review by joshplumley — APPROVED** [review]

**Pull request merged**